.dockerignore

```diff
@@ -1,2 +1,12 @@
-build/
+.git/
 .github/
+build/
+.editorconfig
+.gitattributes
+.gitignore
+CHANGELOG.md
+CODE_OF_CONDUCT.md
+deploy.sh
+font-selection.json
+README.md
+Vagrantfile
```

Comment:
nitpick: can you update your IDE so that it adds a line break to the end of each file. It's rare that it affects us, so this is more about the ugly <img width="27" alt="image" src="https://user-images.githubusercontent.com/94555501/171005791-c316401c-104d-4d34-80c4-d364872ec69c.png"> icon.